inference-engine/thirdparty/clDNN/src/graph_optimizer/remove_redundant_reorders.cpp

```diff
@@ -298,6 +311,19 @@ void remove_redundant_reorders::run(program& p) {
             if (!lo.can_fuse_reorder_to_prev(input, *node.get_users().front(), input.get_output_layout().format, output_layout.format))
                 continue;
 
+            bool skip = false;
+            for (auto& user : node.get_dependency(0).get_users()) {
+                if (user != &node && user->can_be_optimized() &&
+                node.get_output_layout().data_padding.lower_size() != (tensor)0 &&
+                node.get_output_layout().data_padding.upper_size() != (tensor)0) {
```

Comment:
I think you should skip optimization in case of **any** padded sizes **not only both** in the same time